pkgs/development/python-modules/python-openid/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, isPy27
+, nose
+}:
+
+buildPythonPackage rec {
+  pname = "python-openid";
+  version = "2.2.5";
+  disabled = !isPy27;
+
+  src = fetchFromGitHub {
+    owner = "openid";
+    repo = pname;
+    rev = version;
+    sha256 = "198800ka2xyjz4yi2zm23l2a0knmqzxb4wj1nhqps70jz8mwg5mz";
+  };
+
+  checkInputs = [
+    nose
+  ];
+
+  checkPhase = ''
+    nosetests
+  '';
+
+  # most tests require network access
+  doCheck = false;
```

Comment:
you can do `nosetests -I=file_that_needs_networking` to ignore specific files, would still like to see at least a few tests to ensure teh package is roughly usable.

Reply:
This module was completely removed as it was just adding python2Packages.python-openid which is not needed for this python3Packages.apache-airflow.